CONTRIBUTING.md

```diff
@@ -20,33 +20,33 @@ and test the code, and deploy a new version.
 ## Project structure
 
 The business logic for this package is located
-in [`hakai_api/Client.py`](hakai_api/Client.py).
+in [`src/hakai_api/Client.py`](src/hakai_api/Client.py).
 All tests are located in the `tests/` directory.
 
 ## Configuration
 
-### Poetry
+### UV
```

Comment:
nit: I do see that astral typically refers to `uv` in lowercase.

Reply:
Done!